feat(meetup): add Meetup Rust Lyon (#419)
BedrockStreaming/tech.bedrockstreaming.com: CONTRIBUTING.md

- [How to add simple Diagrams with _Mermaids_?](https://sylhare.github.io/Type-on-Strap/2019/11/02/Tech-stuff-example.html#mermaid)
  Mermaid is a really powerful tool to generate Diagram dynamically with some text.
  Check [Mermaid documentation](https://mermaid-js.github.io/mermaid/#/).
- How to add an author? 
  Edit `_data/authors.yml` file to add an author directly in the Yaml file.

>   Author could have a name, an URL and an avatar (which could be a distant file or an image hosted in _images/author_ directory).

Reviewer: ```suggestion
  Authors could have a `name`, an `url` and an `avatar` (which could be a distant file or an image hosted in the `images/avatar` directory).
```

## Add an LFT replay

1. Create a file in `__post` folder name matching this format `YYYY-MM-DD-slug-of-your-article.md`
    Use the date the talk was first given in public.
2. Add the configuration of metadatas at the begining of this file
    > :warning: **To make your videos appear in either `Last Friday Talks`page, tag your post with `lft`.**
    ```markdown
    ---
    layout: video
    # Unique Id of the youtube video clip
    youtubeId: $$$$$$$ 
    # Title of the article
    title: Title of your article

>     # Description of the page (for SEO and context purpose

Reviewer: ```suggestion
    # Description of the page (for SEO and context purpose)
```

  author: conference_speaker 
  # Public event name
  eventName: ******
  # Url to redirect to the event site (optional)
  eventUrl: ******
  # Youtube video id (optional)
  youtubeId: ******
  # Slideshare presentation key (from iframe integration) (optional)
  slideshareKey: ******
  # Bedrock sponsored the event? (default: false)
  sponsored: true
  # Bedrock hosted the event? (default: false)
  hosted: true
```

> It's all folks! Your conference will be displayed in "Meetups & Conferences" page. 

Reviewer: ```suggestion
That's all folks! Your conference will be displayed in "Meetups & Conferences" page. 
```
[That's all folks!](https://p4.storage.canalblog.com/48/63/1204697/126367524_o.png )
